common/src/main/java/org/jboss/gm/common/utils/OTELUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.jboss.gm.common.utils;
+
+import lombok.experimental.UtilityClass;
+
+import org.gradle.api.logging.LogLevel;
+import org.gradle.api.logging.Logger;
+import org.jboss.gm.common.logging.FilteringCustomLogger;
+import org.jboss.gm.common.logging.GMLogger;
+
+import com.redhat.resilience.otel.OTelCLIHelper;
+
+@UtilityClass
+public class OTELUtils {
+
+    private final Logger logger = GMLogger.getLogger(OTELUtils.class);
+
+    private final LogLevel originalLevel = FilteringCustomLogger.getContext().getLevel();
+
+    public void startOTel() {
+        String endpoint = System.getenv("OTEL_EXPORTER_OTLP_ENDPOINT");
+        String service = System.getenv("OTEL_SERVICE_NAME");
+
+        if (endpoint != null) {
+            if (service == null) {
+                service = "gradle-manipulator";
+            }
+            logger.info("Enabling OpenTelemetry collection on {} with service name {} level {}", endpoint, service,
+                    originalLevel);
+            try {
+                if (originalLevel != LogLevel.DEBUG) {
+                    FilteringCustomLogger.getContext().setLevel(LogLevel.DEBUG);
+                }
```

Comment:
The log message seems a little confusing to me as it prints `level originalLevel` in the log, but actually forces it to use `DEBUG`?

Reply:
I was trying to always get the output from the opentelemetry-java-cli library as its useful to see the trace and span ids.

Reply:
I've removed the confusing bit of the logging statement (left over debug that I missed)